app/lib/project/ImportController.js

```diff
@@ -75,8 +76,11 @@ define([
                     return true;
                 }.bind( null, this._master )
             };
-            this._sourceGlyphSet = this._project.getGlyphSet(
```

Comment:
Before pulling this, how to proceed with the issue described here? Should I file an issue and assign you?